src/gui/qgsmaptoolcapture.h

```diff
@@ -88,8 +88,49 @@ class GUI_EXPORT QgsMapToolCapture : public QgsMapToolAdvancedDigitizing
 
 
   protected:
-    int nextPoint( const QgsPoint& mapPoint, QgsPoint& layerPoint );
-    int nextPoint( const QPoint &p, QgsPoint &layerPoint, QgsPoint &mapPoint );
+    /** Converts a map point to layer coordinates
+        @param mapPoint the point in map coordinates
+        @param[inout] layerPoint the point in layer coordinates
+        @return
+          0 in case of success
+          1 if the current layer is null or not a vector layer
+          2 if the transformation failed
+        @deprecated use nextPoint(const QgsPointV2&, QgsPointV2&)
+    */
+    Q_DECL_DEPRECATED int nextPoint( const QgsPoint& mapPoint, QgsPoint& layerPoint );
+
+    /** Converts a map point to layer coordinates
+        @param mapPoint the point in map coordinates
+        @param[inout] layerPoint the point in layer coordinates
+        @return
+          0 in case of success
+          1 if the current layer is null or not a vector layer
+          2 if the transformation failed
+    */
+    int nextPoint( const QgsPointV2& mapPoint, QgsPointV2& layerPoint );
```

Comment:
Also, as per the vector layer edit comment, lets fix these return values and make them return a result enum value